src/ServiceControl.TransportAdapter/ServiceControl.TransportAdapter.csproj

```diff
@@ -38,13 +38,19 @@
   <PropertyGroup>
     <StartupObject />
   </PropertyGroup>
+  <PropertyGroup>
+    <SignAssembly>true</SignAssembly>
+  </PropertyGroup>
+  <PropertyGroup>
+    <AssemblyOriginatorKeyFile>NServiceBus.snk</AssemblyOriginatorKeyFile>
```

Comment:
Yeah, it's there but github collapses it because it is binary. I've moved it one level up.